WordPress/Classes/Services/PageLayoutService.swift

```diff
@@ -66,6 +71,17 @@ class PageLayoutService {
     }
 
     private static let scale = UIScreen.main.nativeScale
+
+    private static let type = "mobile"
+
+    private static let isBeta: String = {
+        let isDevMode = BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
+        if isDevMode == true {
+            return "true"
+        }
+        return "false"
+    }()
```

Comment:
Since the `~=` [does not return an optional](https://github.com/wordpress-mobile/WordPress-iOS/blob/50e1647c52304eef6b84fd330729d03be9796040/WordPress/Classes/Utility/BuildInformation/BuildConfiguration.swift#L26) I would simplify this like:
```
private static let isBeta = String(BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest])
```
This might be opinionated and your solution is probably better since it is easier to see the actual values passed.

Reply:
Great idea. Thanks for the suggestion to simplify it.